plugin/cypress.config.ts

```diff
@@ -14,13 +14,21 @@ export default defineConfig({
   requestTimeout: 15000,
   responseTimeout: 15000,
   fixturesFolder: 'cypress/fixtures',
+  chromeWebSecurity: false,
+  screenshotsFolder: "results/screenshots",
+  videosFolder: "results/videos",
+  // videoUploadOnPasses: false, // this is not supported in cypress 13+, TODO add this back to config
+
   env: {
+    USERNAME: "jenkins", // default value for jenkins
```

Comment:
It is not consistent with username variable in the rest of the configuration (`OC_CLUSTER_USER`). In my opinion I think it is more simple to set just `OC_USERNAME` and `OC_PASSWORD` variable names.

Reply:
I was trying to set safe default not to interfere with other environment variables I could think of, like `OC_USERNAME` for `oc` cli tool or so. I would say we can definitely something more reasonable like `OC_CONSOLE_USER` and `OC_CONSOLE_PASS` or similar. If we merge this, please feel free to raise this as an issue or refactoring improvement

Reply:
variable name changed